pipeline/unified/channel/sign-release-package-linux.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+parameters:
+    unsignedPipelineResource: null
+    unsignedArtifactName: null
+    signedArtifactName: null
+
+jobs:
+    - job: ${{ parameters.signedArtifactName }}
+      pool:
+          vmImage: 'ubuntu-16.04'
+      steps:
+          - template: ../../install-node-prerequisites.yaml
+
+          - task: DownloadPipelineArtifact@2
+            inputs:
+                source: 'specific'
+                runVersion: 'specific'
+                project: $(resources.pipeline.${{ parameters.unsignedPipelineResource }}.projectID)
+                pipeline: $(resources.pipeline.${{ parameters.unsignedPipelineResource }}.pipelineID)
+                runId: $(resources.pipeline.${{ parameters.unsignedPipelineResource }}.runID)
+                artifact: ${{ parameters.unsignedArtifactName }}
+                path: '$(System.DefaultWorkingDirectory)/signing-in-progress/${{ parameters.signedArtifactName }}'
+
+          - task: CopyFiles@2
+            displayName: 'Copy the AppImage file to detachedSignature'
+            inputs:
+                SourceFolder: '$(System.DefaultWorkingDirectory)/signing-in-progress/${{ parameters.signedArtifactName }}'
+                contents: |
+                    Accessibility Insights for Android*.*
+                TargetFolder: '$(System.DefaultWorkingDirectory)/detachedSignature'
+
+          - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+            displayName: 'sign dist/detachedSignature'
+            inputs:
+                ConnectedServiceName: 'ESRP Code Signing'
+                FolderPath: '$(System.DefaultWorkingDirectory)/detachedSignature'
+                Pattern: '*.AppImage'
+                signConfigType: inlineSignParams
+                inlineOperation: |
+                    [
+                        {
+                            "keyCode": "CP-450779-Pgp",
+                            "operationSetCode": "LinuxSign",
+                            "parameters": [],
+                            "toolName": "sign",
+                            "toolVersion": "1.0"
+                        }
+                    ]
+
+          - script: 'rename "s/\.AppImage$/.sig/" $(System.DefaultWorkingDirectory)/detachedSignature/*.AppImage'
```

Comment:
use of `rename` instead of earlier `mv` because source file name differs between channels (Used to explicitly reference `Accessibility Insights for Android.AppImage`, but the file might be `Accessibility Insights for Android - Insider.AppImage`, for instance)